features/adres.feature

```diff
@@ -141,21 +144,17 @@ Functionaliteit: Adresvelden vullen
   Rule: Voor een buitenlands adres worden de adresregels en land gevuld uit de gegevens over verblijf buitenland (groep 13)
 
     Scenario: persoon verblijft in het buitenland
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 999993483 |
-      En de persoon heeft de volgende 'verblijfplaats' gegevens
+    Gegeven de persoon met burgerservicenummer '000000346' heeft de volgende 'verblijfplaats' gegevens
       | naam                              | waarde          |
       | gemeente van inschrijving (09.10) | 1999            |
-      | land adres buitenland (13.10)     | 5010            |
-      | land.omschrijving                 | België          |
+      | land (13.10)                      | 5010            |
       | regel 1 adres buitenland (13.30)  | Rue du pomme 25 |
       | regel 2 adres buitenland (13.40)  | Bruxelles       |
       | regel 3 adres buitenland (13.50)  | postcode 1000   |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                                                                                   |
       | type                | RaadpleegMetBurgerservicenummer                                                          |
-      | burgerservicenummer | 999993483                                                                                |
+      | burgerservicenummer | 000000346                                                                                |
       | fields              | adressering.adresregel1,adressering.adresregel2,adressering.adresregel3,adressering.land |
       Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
       | naam              | waarde          |
```

Comment:
Klopt het dat in de response van een buitenlandsadres adresregel2 niet geconverteerd wordt naar hoofdletters?
Dat zou ik me voor kunnen stellen als hier ook andere dingen kunnen worden opgenomen dan de woonplaats maar ik stel de vraag toch maar even.